app/src/main/res/values-hi/strings.xml

```diff
@@ -380,6 +380,8 @@
         <item>ऋण खाते के लिए भुगतान करने के लिए, खाता अनुभाग पर जाएं, ऋण चुनें और आवश्यक ऋण खाता खोलें
             और भुगतान करें विकल्प पर क्लिक करें।</item>
     </string-array>
+    <string name="cancel_transfer">क्या आप वाकई हस्तांतरण रद्द करना चाहते हैं?</string>
```

Comment:
@luckyman000786  please add the string in all strings.xml, whenever you add a new string. Here is a medium blog(https://medium.com/@therajanmaurya/add-localization-string-in-application-supported-all-languages-using-keyboard-shortcut-807e0a388672). I have written, have a look